src/genie/libs/parser/iosxr/show_vrf.py

```diff
@@ -68,7 +68,7 @@ def cli(self, vrf='', output=None):
 
             # VRF VRF1; RD 200:1; VPN ID not set
             # RD 172.16.94.0:1000
-            p1 = re.compile(r'^VRF +(?P<vrf>[\w\-]+); +'
+            p1 = re.compile(r'^VRF +(?P<vrf>[\w\-\:]+); +'
```

Comment:
Please add a new test to cover your changes. Details can be found [here](https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#folder-based-testing-asa-ios-and-iosxe)

Reply:
Test has been added!

Reply:
Hi @aterkuci I don't see it in the PR's changed files